src/components/Apps/AccountListTable/const.js

```diff
@@ -223,7 +223,7 @@ export const accountQuickFilters = (vm) => [
       {
         label: vm.$t('NoLoginLongTime'),
         filter: {
-          risk: 'long_time_no_login'
+          long_time_no_login: 'true'
         }
       },
       {
```

Comment:
There aren't any clear differences between these two versions of the code you shared. Both seem to be correctly using variables with underscores (_) in them instead of camelCase notation (`$t`, `risk`) which is more readable than plain underscored strings.

However, if there's a reason why this should not be done, please let me know so I can help clarify further on why it would need to be changed. Without that information, though, both codes are functioning as intended according to their context. For general advice, however, avoiding unnecessary special characters like underscores and capitalizing variable names can improve readability and maintainability when debugging or updating your code later.